src/ts/errorutils.ts

```diff
@@ -67,7 +68,11 @@ export namespace ErrorUtils {
     3100: 'An Advertising module error has occurred. Refer to the attached AdvertisingError.',
   };
 
-  export const defaultErrorMessageTranslator: ErrorMessageTranslator = (error: ErrorEvent) => {
+  export const defaultMobileV3ErrorMessageTranslator = (error: MobileV3PlayerErrorEvent | MobileV3SourceErrorEvent) => {
+    return `${error.message}\n(${error.name})`;
```

Comment:
There is no property `name` on the v3 `ErrorEvent`:

![IMG_0010 1](https://user-images.githubusercontent.com/6216959/109941906-ebbe6800-7cd3-11eb-8d39-49ebe1acb160.PNG)